src/osd/PrimaryLogPG.cc

```diff
@@ -10492,6 +10496,9 @@ int PrimaryLogPG::start_dedup(OpRequestRef op, ObjectContextRef obc)
   for (auto p : chunks) {
     hobject_t target = mop->new_manifest.chunk_map[p.first].oid;
     if (refs.find(target) == refs.end()) {
+      if (target.get_hash() == 0) {
+        return -EINVAL;
+      }
```

Comment:
This change is not necessary. What is the purpose of this?

Reply:
This change should have been removed in advance, but it was my mistake.
Sorry for confusing you.